Sources/Core/API/Actor.swift

```diff
@@ -83,7 +83,13 @@ public final class Actor: SceneObject, InstanceHashable, ActionPerformer,
 
     /// Initialize an instance of this class
     public init() {}
-
+    
+    // MARK: - SceneObject
+    
+    func addLayer(to superlayer: Layer) {
```

Comment:
Nitpicky detail: I usually mark functions explicitly as `internal`. While it's not required, it makes it a lot easier to see that a certain function is intended just for internal use, that the author made that decision - which makes it easier for other people reading the code 🙂

Reply:
Good point! I will fix it!